src/bootstrap/tool.rs

```diff
@@ -742,7 +742,7 @@ pub struct RustAnalyzerProcMacroSrv {
 impl Step for RustAnalyzerProcMacroSrv {
     type Output = Option<PathBuf>;
     const DEFAULT: bool = true;
-    const ONLY_HOSTS: bool = false;
+    const ONLY_HOSTS: bool = true;
```

Comment:
I'm not sure this is right. Isn't this libproc_macro but for rust-analyzer? IOW, shouldn't it be shipped whenever we ship std?

Reply:
No, we should ship this wherever we ship a host compiler. The binary provides a stable interface between `rust-analyzer` and the current compiler's proc macro implementation, so there should be no point in shipping it outside of host toolchains.